Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,58 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+User Applications can make use of the NFS client running on the external ceph standalone cluster.
+For that user needs to:
+
+i) Export the nfs client to a particular cephfs filesystem.
+
+ii) Create a PV, using the `nfs-client-server-ip`.
+
+iii) And then create a PVC using the above PV.
+
+And the PVC can be used by user applications to utilize the nfs type storage.
+
+Procedure:
+
+Create the export:
+
+```yaml
+ceph nfs export create cephfs <nfs-client-name> /test <filesystem-name>
+```
+
+Create the PV:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: <pv-name>
+spec:
+  capacity:
+    storage: <storage_size>
+  accessModes:
+  - ReadWriteOnce
+  nfs:
+    path: /test
+    server: <nfs-client-server-ip>
```

Comment:
kubernetes is removing intree drivers. this might get deprecated/removed in favor of CSI, please check

Reply:
not able to find anything specific to nfs https://kubernetes.io/blog/2022/11/18/upcoming-changes-in-kubernetes-1-26/

Anyway if it need migration we can updated the steps later.

Reply:
correcting myself. nfs, hostpath, and a few are not deprecated/removed.